multipaz-zkp/src/commonMain/kotlin/org/multipaz/zkp/util/Util.kt

```diff
@@ -0,0 +1,25 @@
+package org.multipaz.zkp.util
+
+import kotlinx.datetime.LocalDateTime
+
+/**
+ * Utility class for cryptographic operations.
+ * This class provides utility methods for common cryptographic operations used in ZKP.
+ */
+object Util {
+    /**
+     * Formats a date as an ISO 8601 string.
+     *
+     * @return The formatted string
+     */
+    fun getAsFormattedDateString(date: LocalDateTime): String {
+        val year = date.year.toString().padStart(4, '0')
+        val month = date.monthNumber.toString().padStart(2, '0')
+        val day = date.dayOfMonth.toString().padStart(2, '0')
+        val hour = date.hour.toString().padStart(2, '0')
+        val minute = date.minute.toString().padStart(2, '0')
+        val second = date.second.toString().padStart(2, '0')
+
+        return "$year-$month-${day}T$hour:$minute:${second}Z"
+    }
+}
```

Comment:
I think `LocalDateTime.format(LocalDateTime.Formats.ISO)` does this?

Otherwise, this should go into dateTimeUtils.kt instead of here.